packages/aws-cdk/lib/init-templates/app/python/hello/hello_stack.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+from aws_cdk import (
+    aws_iam as iam,
+    aws_sqs as sqs,
+    aws_sns as sns,
+    cdk
+)
+
+from hello_construct import HelloConstruct
+
+class MyStack(cdk.Stack):
+
+    def __init__(self, app: cdk.App, id: str) -> None:
+        super().__init__(app, id)
+
+        queue = sqs.Queue(
+            self,
+            "MyFirstQueue",
+            visibility_timeout_sec=300,
+        )
+
+        topic = sns.Topic(
+            self,
+            "MyFirstTopic",
+            display_name="My First Topic"
+        )
+
+        topic.subscribe_queue(queue)
+
+        hello = HelloConstruct(self, "foobar", num_buckets=4)
+        user = iam.User(self, "MyUser")
```

Comment:
add some comments on what this does under the hood (given it's a sample app, it could be educational).